frontend/src/features/public-form/utils/inputTransformation.ts

```diff
@@ -214,7 +214,7 @@ const transformToSignatureOutput = (
   input?: SignatureFieldValues | SignatureFieldResponseV3,
 ): SignatureResponse => {
   let answerArray: string[] = []
-  if (input !== undefined && input !== null) {
+  if (input && input.value !== undefined && input.value !== null) {
```

Comment:
Hmm, seems like the TS is failing somewhere - cause based on the SingatureFieldValues type, the `value` should always be defined